Assets/HoloToolkit/Sharing/Tests/Scripts/RemoteHeadManager.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 using HoloToolkit.Unity;
+//using UnityEditor;
```

Comment:
I agree. If you comment out code, there should be a comment stating why. If there is no reason for commenting it out, then just plain out delete it. This helps to keep the code base clean and unambiguous as commented out code can present ambiguous meanings.